config/ufs/machines/config_machines.xml

```diff
@@ -451,7 +451,66 @@ This allows using a different mpirun command to launch unit tests
       <env name="W3NCO_LIBd">$ENV{NCEPLIBS_DIR}/lib/libw3nco_v2.0.6_d.a</env>
     </environment_variables>
   </machine>
-
+ <machine MACH="gaea">
+    <DESC>
+      port to gaea
+    </DESC>
+    <NODENAME_REGEX>gaea*</NODENAME_REGEX>
+    <OS>CNL</OS>
+    <COMPILERS>intel</COMPILERS>
+    <MPILIBS>mpt</MPILIBS>
+    <CHARGE_ACCOUNT>esrl_bmcs</CHARGE_ACCOUNT>
+    <SAVE_TIMING_DIR> </SAVE_TIMING_DIR>
+    <CIME_OUTPUT_ROOT>$ENV{SCRATCH}/ufs-mrweather-app</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>/lustre/f2/pdata/esrl/gsd/ufs/ufs-release-v1/CIME_UFSINPUT</DIN_LOC_ROOT>
+    <DIN_LOC_ROOT_CLMFORC>${DIN_LOC_ROOT}/lmwg</DIN_LOC_ROOT_CLMFORC>
+    <DOUT_S_ROOT>${CIME_OUTPUT_ROOT}/ufs-runs/archive/$CASE</DOUT_S_ROOT>
+    <BASELINE_ROOT>${CIME_OUTPUT_ROOT}/ufs-runs/baselines</BASELINE_ROOT>
+    <CCSM_CPRNC>${CIME_OUTPUT_ROOT}/ufs-runs/tools/cprnc/cprnc</CCSM_CPRNC>
+    <GMAKE>make</GMAKE>
+    <GMAKE_J>8</GMAKE_J>
+    <BATCH_SYSTEM>slurm</BATCH_SYSTEM>
+    <SUPPORTED_BY>NOAA</SUPPORTED_BY>
+    <MAX_TASKS_PER_NODE>16</MAX_TASKS_PER_NODE>
+    <MAX_MPITASKS_PER_NODE>16</MAX_MPITASKS_PER_NODE>
+    <PROJECT_REQUIRED>FALSE</PROJECT_REQUIRED>
+    <mpirun mpilib="default">
+     <executable>srun</executable>
+      <arguments>
+        <arg name="label"> --label</arg>
+        <arg name="num_tasks"> -n {{ total_tasks }}</arg>
+    </arguments>
+    </mpirun>
+    <module_system type="module">
+      <init_path lang="perl">/opt/cray/pe/modules/3.2.10.5/init/perl.pm</init_path>
+      <init_path lang="python">/opt/cray/pe/modules/3.2.10.5/init/python.py</init_path>
+      <init_path lang="csh">/opt/cray/pe/modules/3.2.10.5/init/csh</init_path>
+      <init_path lang="sh">/opt/cray/pe/modules/3.2.10.5/init/sh</init_path>
+      <cmd_path lang="perl">/opt/cray/pe/modules/4.1.3.1/bin/modulecmd perl</cmd_path>
+      <cmd_path lang="python">/opt/cray/pe/modules/4.1.3.1/bin/modulecmd python</cmd_path>
+      <cmd_path lang="sh">module</cmd_path>
+      <cmd_path lang="csh">module</cmd_path>
+      <modules compiler="intel">
+        <command name="load">PrgEnv-intel/6.0.3</command>
+        <command name="load">intel/18.0.3.222</command>
+        <command name="use">/lustre/f2/pdata/esrl/gsd/ufs/modules/modulefiles/generic</command>
+        <command name="load">cmake/3.16.4</command>
+        <command name="use">/lustre/f2/pdata/esrl/gsd/ufs/modules/modulefiles/intel-18.0.3.222</command>
+        <command name="load">NCEPlibs/1.0.0.beta03</command>
+      </modules>
+    </module_system>
+    <environment_variables>
+      <env name="OMP_STACKSIZE">256M</env>
+      <env name="NCEP_LIBS">$ENV{NCEPLIBS_DIR}</env>
+      <env name="SCRATCH">/lustre/f2/scratch</env>
```

Comment:
Since you are setting SCRATCH here I suggest that we change it here and above to UFS_SCRATCH to be consistant with other systems.  Also for consistancy perhaps we should require users to define UFS_SCRATCH before using cime?